pages/seo.js

```diff
@@ -1,15 +1,120 @@
 import React from "react";
 import Head from "next/head";
 
-const Seo = ({ title, description, url, date, article }) => {
+const Seo = ({
+  title,
+  description,
+  authorName,
+  url,
+  image_url,
+  publishedAt,
+  publishedTime,
+  readingTime,
+  article,
+}) => {
   return (
     <Head>
-      <meta charset="UTF-8" />
```

Comment:
Charset, robots and viewport are mandatory headers for any website. Let's not remove it. Also let's add meta properties for home page also. Refer ticket for content